src/transformStateWithClones.js

```diff
@@ -66,6 +66,44 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  let result = {};
+
+  result = Object.assign(result, state);
+
+  const massResult = [];
+
+  for (let i = 0; i < transforms.length; i++) {
+    const obj = transforms[i];
+
+    if (obj.operation === 'addProperties') {
+      const object = obj.properties;
+
+      for (const key in object) {
+        result[key] = object[key];
+      }
+      massResult.push({ ...result });
```

Comment:
You push result 3 times.Do this at the end of the for

Reply:
agree, move push to the end of the for loop